howso/marginal.amlg

```diff
@@ -790,23 +770,23 @@
 								))
 							)
 
-							;output assoc of marginal stats for the feature, replace .nan with null if needed
+							;output assoc of marginal stats for the feature
 							(assoc
```

Comment:
not saying you need do this, but this is a fitting place to use a { } style for the assoc, imo